.gitignore

```diff
@@ -30,3 +30,12 @@
 
 # Debug files
 *.dSYM/
+.project
+CMakeCache.txt
+CMakeFiles
+Makefile
+cmake_install.cmake
+tinyiiod-test
+build/
+ci/travis/buil
```

Comment:
`ci/travis/buil` looks like a typo;

also here [since this is changing] you could group them with a note
```
# cmake & build artifacts
.project
CMakeCache.txt
CMakeFiles
Makefile
cmake_install.cmake
tinyiiod-test
build/
ci/travis/build
*.orig
```

Reply:
All ignored files are grouped mow